src/earn/EarnCollectScreen.tsx

```diff
@@ -132,6 +133,24 @@ export default function EarnCollectScreen({ route }: Props) {
             style={styles.error}
           />
         )}
+        {asyncPreparedTransactions.result?.type === 'not-enough-balance-for-gas' && (
+          <InLineNotification
+            variant={NotificationVariant.Warning}
+            title={t('earnFlow.collect.noGasTitle', { symbol: feeCurrencies[0].symbol })}
+            description={t('earnFlow.collect.noGasDescription', {
+              symbol: feeCurrencies[0].symbol,
+              network: NETWORK_NAMES[depositToken.networkId],
+            })}
+            ctaLabel={t('earnFlow.collect.noGasCta', {
+              symbol: feeCurrencies[0].symbol,
+              network: NETWORK_NAMES[depositToken.networkId],
+            })}
+            onPressCta={() => {
+              // navigate
```

Comment:
Looks like navigate should go to `FiatExchangeAmount` similar to [here](https://github.com/valora-inc/wallet/blob/c55cd0ac26f892bdc6da3fd26ab46b326b63aab9/src/tokens/TokenDetails.tsx#L216)